modules/prototype-kit/README.md

```diff
@@ -0,0 +1,35 @@
+# GOV.UK Prototype Kit
+
+Go to the [GOV.UK Prototype Kit site](https://govuk-prototype-kit.herokuapp.com/docs) to download the latest version and read the documentation.
+
+## About the Prototype Kit
+
+The Prototype Kit provides a simple way to make interactive prototypes that look like pages on GOV.UK. These prototypes can be used to show ideas to people you work with, and to do user research.
+
+Read the [project principles](https://govuk-prototype-kit.herokuapp.com/docs/principles).
+
+## Security
+
+If you publish your prototypes online, they **must** be protected by a [username and password](https://govuk-prototype-kit.herokuapp.com/docs/publishing-on-heroku). This is to prevent members of the public finding prototypes and thinking they are real services.
```

Comment:
Are we doing this already or do we only need to when we actually start adding some pages/features etc?

Reply:
Not done currently, but can add this, I tend to go with the banner at the top saying this is a prototype. To be fair this hasn't been done on other projects I've worked on.